e2e/cypress/integration/system_console/team_and_channel_assign_roles_spec.js

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('System Console', () => {
+    it('MM-20059 - System Admin can map roles to groups from Team Configuration screen', () => {
+        cy.apiLogin('sysadmin');
+
+        // # Go to system admin page and to team configuration page of channel "eligendi"
+        cy.visit('/admin_console/user_management/teams');
+        cy.findByTestId('eligendiedit').click();
```

Comment:
Instead of hardcoding the team name, please find the first team and then click on it. Currently it fails, as it is not able to find this team on my instance.

Reply:
I didn't account for pagination. Now we search for the team.